book/projects/index.md

```diff
@@ -13,8 +13,8 @@ You can find our comprehensive guide for organizing projects before, during, and
 
 ## List of Projects
 
-Here is our current list of project for our {{dates}} {{ hackweek }} hackweek:
+Projects for the {{dates}} {{ hackweek }} will be listed here during the event. Prior to them being listed here, participants can express their interest in one or more projects on the projects spreadsheet: {{ project_spreadsheet_url }}
```

Comment:
@JessicaS11 The "project_spreadsheet_url" is what I meant by the project list. Though this was something we used for GeoSMART last year, and haven't used this year. I'm not sure if it should still be included in the template or not.

Reply:
> I'm not sure if it should still be included in the template or not.

Sounds like a good discussion topic for a future hw-as-a-service meeting!